impeller/playground/playground.cc

```diff
@@ -248,7 +248,7 @@ bool Playground::OpenPlaygroundHere(
       ImGui::NewFrame();
       bool result = render_callback(render_target);
       ImGui::Render();
-
+      return result;
```

Comment:
was this intentional?

Reply:
This seems like it will artificially make the test pass or behave unexpectedly.

Reply:
Yeah, unintentional. I'll get rid of it before landing.